### PR TITLE
Fix search token

### PR DESCRIPTION
**Pull request by biocom**
# Summary

Addresses #1021 

- Makes the token search modal a bit wider (desktop)
- Introduces text ellipsis (CSS) with a max-width for the text container (to support text-overflow: ellipsis) 
- Mobile responsive

Screen Shot 2021-07-23 at 13 16 47Screen Shot 2021-07-23 at 13 15 57Screen Shot 2021-07-23 at 13 15 36

### CODE REVIEW

File: src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx [hunk 1]
```diff
@@ -25,12 +26,6 @@ import { formatSmart } from 'utils/format'
 import { SHORT_PRECISION } from 'constants/index'
 import { FeeInformationTooltipWrapper } from 'components/swap/FeeInformationTooltip'
 
-export const CurrencySearchModal = styled(CurrencySearchModalUni)`
```
Comment (biocom) [inline src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx:28]: Didn't seem part of the original UNI file. Moved to the mod index.

File: src/custom/components/CurrencyInputPanel/index.tsx [hunk 2]
```diff
@@ -14,12 +14,20 @@ import CurrencyInputPanelMod, {
...
   StyledBalanceMax,
 } from './CurrencyInputPanelMod'
+import CurrencySearchModalUni from '@src/components/SearchModal/CurrencySearchModal'
 import { RowBetween } from 'components/Row'
 import { FeeInformationTooltipWrapper } from 'components/swap/FeeInformationTooltip'
 
 import { StyledLogo } from 'components/CurrencyLogo'
 import { LONG_LOAD_THRESHOLD } from 'constants/index'
 
+export const CurrencySearchModal = styled(CurrencySearchModalUni)`
```
Comment (biocom) [inline src/custom/components/CurrencyInputPanel/index.tsx:24]: Moved the custom code from the MOD file into this index file.

**Comment by elena-zh** [comment]
Changes LGTM!
Minor issues and enhancements: 
1. Not related to this PR, as the issue is reproducible on Stage. However, it might be fixed in this pr, I think. So, tooltip is not visible near the message 'Expanded results.. ' in the dark mode: 
tooltip icon black
tooltip exp

2. When a value in the token list is cropped, It would be nice to show full text when hover a mouse on it.
full text

WDYT?

File: src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx:28]: 👍

File: src/custom/components/CurrencyInputPanel/index.tsx (hunk 2 above)
Comment (W3stside) [reply src/custom/components/CurrencyInputPanel/index.tsx:24]: 👍

**Review by W3stside — APPROVED** [review]
@biocom nice work! I would just addres @elena-zh 's comments and then g2m

**Comment by biocom** [comment]
@elena-zh @W3stside Fixed:
Screen Shot 2021-07-23 at 17 15 33Screen Shot 2021-07-23 at 17 15 21

**Review by W3stside — APPROVED** [review]
noice

**Comment by elena-zh** [comment]
The 1st case LGTM!

Are we going to address  the 2nd case: show full text when hover a mouse on it:
full text
I mean something like this 
image

**Comment by biocom** [comment]
For the second case @elena-zh - I feel a better solution is a *non* tooltip solution. E.g. show everything by making the rows variable in height (with some limitations for really crazy long names etc.).

That solution would solve for both mobile/desktop, given the tooltip isn't such a good solution for mobile (no hover...).

Let me know your thoughts. I think we need a separate ticket to address this case.

**Review by alfetopito — APPROVED** [review]
Looking really good :clap:

**Comment by elena-zh** [comment]
@biocom , thanks!
I've created a separate issue #1051  for this case

**Pull request merged**